raiden/tests/utils/smoketest.py

```diff
@@ -218,27 +150,10 @@ def get_private_key(keystore):
     return accmgr.get_privkey(addresses[0], DEFAULT_PASSPHRASE)
 
 
-@contextmanager
-def setup_testchain_and_raiden(
```

Comment:
This context manager was unnecessary. Now the `ui:smoketest` does this directly.